Source/WebCore/dom/Position.cpp

```diff
@@ -802,50 +811,51 @@ Position Position::downstream(EditingBoundaryCrossingRule rule) const
         return { };
 
     // iterate forward from there, looking for a qualified position
-    Node* boundary = enclosingVisualBoundary(startNode.get());
+    RefPtr boundary = enclosingVisualBoundary(startNode.get());
     // FIXME: PositionIterator should respect Before and After positions.
-    PositionIterator lastVisible = m_anchorType == PositionIsAfterAnchor ? makeDeprecatedLegacyPosition(m_anchorNode.copyRef(), caretMaxOffset(*m_anchorNode)) : *this;
+    auto anchorNode = protectedAnchorNode();
```

Comment:
Same comment as earlier about unnecessary call to a protected function.

Reply:
Changed.